documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan (week beginning...)>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
+
+## ✅ Release Checklist
+
+First steps:
+
+  - [ ] Fork a new branch (`release/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
+  - [ ] Prep the changelog
+  - [ ] Bump the version in `version.go` in the `master` branch to `vX.(Y+1).0-dev`.
+  - [ ] Follow the RC release process to cut the first RC.
+    
+Prepping an RC:
+
+- [ ] version string in `build/version.go` has been updated (in the `release/vX.Y.Z` branch).
+- [ ] tag commit with `vX.Y.Z-rcN`
+- [ ] cut a pre-release on [github](https://github.com/filecoin-project/lotus/releases)
```

Comment:
Can we provide a URL that prepulates the form?  At the minimum we can do https://github.com/filecoin-project/lotus/releases/new?prerelease=true 

Other fields that can be filled in https://github.com/isaacs/github/issues/1410#issuecomment-442240267

What should be specified for the other values in the form?

This is something a script could help do (e.g., by taking in X/Y/Z it can generate more of the URL parameters).

Reply:
This is neat! I grabbed the prerelase param. I think everything else will need to be specific to the version, making a note for next steps.